### PR TITLE
Add Juniper rules

### PR DESCRIPTION
**Pull request by AngelFreak**
Add some very basic Juniper rules from czerwonk/junos_exporter.

### CODE REVIEW

File: _data/rules.yml [hunk 1]
```diff
@@ -360,3 +360,21 @@ services:
...
+        rules:
+        - name: Switch is down
+          description: The switch appears to be down
+          query: junos_up == 0
+          severity: critical
+        - name: High Bandwith Usage 1GiB
+          description: Interfaceis highly saturated for at least 2 min. (> 0.95GiB/s)
+          query: 'irate(junos_interface_transmit_bytes[2m]) * 8 > 9.5e+09'
+          severity: warning
+        - name: High Bandwith Usage 10GiB
```
Comment (samber) [inline _data/rules.yml]: Thanks for your contribution @AngelFreak. I never heard about Juniper before!

I'm not sure it is necessary to duplicate for 10GB and 1GB interfaces.
Or I would prefer a duplication to differentiate criticity: a 95% threshold in critical + 80% threshold in warning.

I think you reversed `9.5e+08` and `9.5e+09` for 10GB and 1GB ifaces. 😄

Can we change `9.5e+08` by `1e+09 * 0.95` to facilitate reading ?

WDYT ?

Comment (AngelFreak) [reply _data/rules.yml]: Thanks for pointing it out, i agree that the notation should be more readable.
I will make the changes tomorrow :)

Comment (AngelFreak) [reply _data/rules.yml]: Will you please verity the new notation and the expected outcome ?
Just to make sure i did not make any calculation errors.

Comment (AngelFreak) [reply _data/rules.yml]: I wonder if it should be 1e+10 for a 10Gbit interface?

Comment (samber) [reply _data/rules.yml]: 1e+09 is for 1GB 😄

File: _data/rules.yml [hunk 2]
```diff
@@ -360,3 +360,18 @@ services:
...
+  - name: Juniper
+    exporters: 
+      - name: czerwonk/junos_exporter
+        doc_url: https://github.com/czerwonk/junos_exporter
+        rules:
+        - name: Switch is down
+          description: The switch appears to be down
+          query: junos_up == 0
+          severity: critical
+          severity: warning
```
Comment (samber) [inline _data/rules.yml]: Only one severity needed here ;)

```
          severity: critical
          severity: warning
```

Comment (AngelFreak) [reply _data/rules.yml]: Woow i'm sorry, it should be fixed now

**Comment by samber** [comment]
Thanks @AngelFreak !

Let's merge that ;)

**Pull request merged**